### PR TITLE
Added string uuid validation.

### PR DESCRIPTION
**Pull request by artdude543**
Added UUID validation for easier validation when working with UUIDs on models.

- I've opted to support versions 3, 4 and 5 due to the validation module used only supports these.
- I'll send a PR to the docs branch also when if this is pulled in.

### CODE REVIEW

**Comment by artdude543** [comment]
Any chance of this being looked at? The failed test does not have anything to do with this PR (If that's why it's not been looked at)

**Comment by neumino** [comment]
Arg sorry I missed that. I'll look into it tonight.

File: lib/type/string.js [hunk 1]
```diff
@@ -251,6 +256,19 @@ TypeString.prototype.default = function(fnOrValue) {
   return this;
 }
 
+/**
+ * Set the string to be a uuid.
+ * @param {number} version
+ * @return {TypeString}
+ */
+TypeString.prototype.uuid = function(version) {
+  if (version < 3 || version > 5) {
```
Comment (neumino) [inline lib/type/string.js]: Can you also check that version is a number?

**Comment by neumino** [comment]
Besides the one little comment, that looks good to me.

**Comment by artdude543** [comment]
I'll add the number check now. Not sure how I actually forgot to add that in the first place xD

**Comment by artdude543** [comment]
Huh, well the tests passed locally O_o gonna check the code again.

**Comment by artdude543** [comment]
I've removed the test for number checking as I'm not sure how to create a test which validates the model. As the error would be thrown from the Model and I'm new to tests so still learning :/

**Comment by neumino** [comment]
Thanks, merged!

**Comment by neumino** [comment]
Can you follow up with a PR on the `gh-pages` branch? Thanks!

**Pull request merged**